### PR TITLE
whats new in celery 5.3.0

### PR DESCRIPTION
**Pull request by auvipy**
work in progress

### CODE REVIEW

File: docs/whatsnew-5.3.rst [hunk 1]
```diff
@@ -1,9 +1,9 @@
-.. _whatsnew-5.2:
+.. _whatsnew-5.3:
 
 =========================================
- What's new in Celery 5.2 (Dawn Chorus)
+ What's new in Celery 5.3 (?)
 =========================================
-:Author: Omer Katz (``omer.drow at gmail.com``)
+:Author: Asif Saif Uddin (``auvipy at gmail.com``), Tomer Nosrati ().
```
Comment (Nusnus) [inline docs/whatsnew-5.3.rst:6]:
```suggestion
:Author: Asif Saif Uddin (``auvipy at gmail.com``), Tomer Nosrati (``tomer.nosrati at gmail.com``).
```

**Comment by auvipy** [comment]
this need more work as you can see

**Comment by auvipy** [comment]
@Nusnus you can take over from here and improve the changelog. and ping me for another review. I have some issues to look after so can't put much effort here right now.

**Comment by Nusnus** [comment]
Np bro!
I got this

**Pull request closed without merging**